src/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+include(GenerateExportHeader)
+
+set(HEADERS matio_private.h mat4.h mat5.h matio.h matioConfig.h)
+#matio_io.c
+set(SRC_FILES mat.c mat4.c mat5.c io.c endian.c matvar_cell.c matvar_struct.c read_data.c snprintf.c)
+if (ZLIB_LIBRARIES)
+    set(SRC_FILES ${SRC_FILES} inflate.c)
+endif()
+
+if (WIN32)
+    set(SRC_FILES ${SRC_FILES} matio.rc)
+endif()
+
+set(FORTRAN_INTERFACE_FILE)
+if (ENABLE_FORTRAN)
+    include(FortranCInterface)
```

Comment:
Fortran interface is unmaintained, see #51.

Reply:
OK. I must say that I have not tested it much recently. If the cmake version can help here, why not. If you prefer to totally forget the fortran interface, I'll happily will remove that code...